packages/web/src/common/store/pages/collection/lineups/sagas.js

```diff
@@ -39,7 +39,7 @@ function* getCollectionTracks() {
   // Some playlists & albums may not have a metadata time, so use the time if not.
   // If for whatever reason, the metadata time is off, prefer the release date
   const times = tracks.map((t) =>
-    Math.max(t.metadata_time || t.time, collection.release_date)
+    Math.max(t.metadata_time || t.time, moment(collection.release_date).unix())
```

Comment:
dayjs pls

Reply:
this file uses moment, we can migrate later. I don't want to change the expected value